package.json

```diff
@@ -23,5 +24,9 @@
   },
   "author": "",
   "license": "MIT",
-  "packageManager": "npm@10.5.0"
+  "packageManager": "npm@10.5.0",
+  "dependencies": {
+    "@requestnetwork/currency": "^0.18.0",
+    "bits-ui": "^0.21.12"
```

Comment:
Are we not using `shacdn/ui` anymore?

Reply:
Svelte shadcn uses bits-ui under the hood.